spec/support/fixtures/user.rb

```diff
@@ -19,4 +19,12 @@ def strange_name
   def superman?
     full_name == "Clark Kent"
   end
+
+  def myself
+    self
+  end
+
+  def other
+    Other.new(text: strange_name)
+  end
```

Comment:
Maybe we can add another method and tests for checking the behavior when the method returns a PORO?

Do you think it makes sense?